android/app/src/main/res/layout/tracksactvity_item.xml

```diff
@@ -105,14 +105,21 @@
 
                     <TextView
                         android:id="@+id/session_speaker"
-                        android:layout_width="wrap_content"
+                        android:layout_width="@dimen/card_width"
                         android:layout_height="wrap_content"
+                        android:layout_gravity="fill"
+                        android:layout_toRightOf="@id/icon_speaker"
                         android:layout_marginLeft="@dimen/layout_margin_large"
-                        android:layout_marginStart="@dimen/layout_margin_large"
                         android:textColor="@color/primary_text"
                         android:textSize="@dimen/text_size_large"
                         tools:text="Speaker" />
-                </LinearLayout>
+                    <TextView
+                        android:layout_width="wrap_content"
+                        android:layout_height="wrap_content"
+                        android:id="@+id/session_status"
+                        android:layout_alignParentRight="true"
+                        android:textStyle="bold" />
+                </RelativeLayout>
```

Comment:
Could you also explain the reason behind doing that please. I don't get why we need it 😅